spec/factories/petition_plugin/details.rb

```diff
@@ -1,13 +1,13 @@
+
 FactoryGirl.define do
-  factory :petition_plugin_information, class: 'PetitionPlugin::Information' do
+  factory :petition_plugin_detail, class: 'PetitionPlugin::Detail' do
     plugin_relation do
       cycle = CycleTestHelper.create_cycle_with_phase(phases: [{ plugin_type: :petition }])
       cycle.plugin_relations.first
     end
     call_to_action 'Assine o projeto'
     signatures_required 4000
     presentation 'Apresentação'
-    body '== Projeto de lei'
-    document_url 'http://projeto.bla'
+    petition_detail_versions { [ FactoryGirl.build(:petition_plugin_detail_version) ] }
```

Comment:
I think you could use a trait and avoid creating a version when maybe you do not need.

```ruby
trait :with_version do
  petition_detail_versions { [ FactoryGirl.build(:petition_plugin_detail_version) ] }
end
```